components/business-adaptors/sap/src/main/java/org/wso2/carbon/transports/sap/SAPTransportSender.java

```diff
@@ -168,29 +180,24 @@ public void sendMessage(MessageContext messageContext, String targetEPR,
                     OMElement payLoad,body;
                     body = messageContext.getEnvelope().getBody();
                     payLoad = body.getFirstChildWithName(new QName(RFCConstants.BAPIRFC));
-                    if (log.isDebugEnabled()){
-                        log.debug("Received RFC/Meta DATA: " + payLoad);
-                    }
-                    String rfcFunctionName = RFCMetaDataParser.getBAPIRFCFucntionName(payLoad);
+                    log.info("Received RFC/Meta DATA: " + payLoad);
```

Comment:
Should this be an INFO level log? At each sendMessage this will be executed.

Reply:
it should be a debug log.

Reply:
done